### ISSUE TITLE
Re size the popup

### ISSUE THREAD

**Issue by jurate911** [Esri/angular-esri-map#32]
In the sample Webmap  you have a popup available, but if  you re size the application the map responds to different screen size but the popup looks like has a fixed size. Is there a way to code the popup that it would also reside based on the screen change.

**Comment by tomwayson** [comment]
I'm not aware of any configuration that you can set on the map or the pop up that would make it responsive to changes in the screen size

@alaframboise may know of a better way, but you could basically code this yourself by listening for changes to the size of the map or it's container and then calling the [popup's resize method](https://developers.arcgis.com/javascript/jsapi/popup-amd.html#resize).

HTH

**Comment by alaframboise** [comment]
The popup will try to reposition itself automatically https://developers.arcgis.com/javascript/jsapi/popup-amd.html#popup1 but if that's not the behavior you are after take a look at the code in https://github.com/Esri/bootstrap-map-js.

**Comment by tomwayson** [comment]
FYI - the docs app for this repo does not include the bootstap-map-js CSS even though it's using Bootstrap. I was going to add it, but didn't see any issues w/ the (very simple) examples, so I decided to leave it out for clarity's sake.

Your own app *may* need at least the CSS from https://github.com/Esri/bootstrap-map-js if you are using Bootstrap. 

Closing as this is not related to the code in this repo.
